layouts/millennium_overview_sidebar.html

```diff
@@ -0,0 +1,17 @@
+<div id="js-sidebar" class="sidebar-shell">
+  <div class="js-toggle-list sidebar-module expandable">
+    <ul>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/overview/">Overview</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/authorization/">Authorization</a></li>
+        </ul>
+      </li>
+    </ul>
+  </div> <!-- /sidebar-module -->
+  <div class="sidebar-module notice">
+    <p>This website is a <a href="https://github.com/cerner/fhir.cerner.com" target="_blank">public GitHub
+      repository</a>. Please help us by forking the project and adding to it.</p>
+  </div>
+</div><!-- /sidebar-shell -->
```

Comment:
add newline to end of file

Reply:
Fixed here: https://github.com/cerner/fhir.cerner.com/commit/c80e789fa7f0d4172d8d6576c7669c60149b65c1